core/common/src/main/java/alluxio/metrics/MetricsConfig.java

```diff
@@ -178,4 +112,23 @@ public void setServletProperties() {
     }
     return subProperties;
   }
+
+  /**
+   * This method removes the instance prefix in the properties. This is to make the configuration
+   * parsing logic backward compatible with old configuration format.
+   */
```

Comment:
The old format can still be parsed. There is logic in MetricsConfig (removeInstancePrefix) to deal with this